packages/language-tools/src/util/extractor.ts

```diff
@@ -161,12 +163,12 @@ export class Extracted {
  * on the view type.
  */
 abstract class TokenView {
-  #tokens: Token[];
+  tokens: Token[];
```

Comment:
This is an awful hack so that I can generate Volar `CodeMapping` objects from Tokens.

A better solution would be to have the extractor be able to generate `CodeMapping`s itself...